lntest/itest/temp_lnd_test.go

```diff
@@ -55,9 +55,9 @@ var (
 // TestLightningNetworkDaemonTemp performs a series of integration tests
 // amongst a programmatically driven network of lnd nodes.
 func TestLightningNetworkDaemonTemp(t *testing.T) {
-	if !*tempTest {
-		t.Skip("Running old tests, new tests are skipped")
-	}
+	// if !*tempTest {
```

Comment:
If you comment this out, all tests are run in the "old" tests. You can run a specific test case of the new tests with `make itest icase=xyz temptest=true`.

Reply:
I accidentally committed this